transform/models/intermediate/imputation/int_imputation__five_minute_aggregation.sql

```diff
@@ -113,6 +113,7 @@ samples_requiring_imputation_with_coeffs as (
     asof join coeffs  -- noqa
         match_condition(samples_requiring_imputation.sample_date >= coeffs.regression_date)  -- noqa
         on samples_requiring_imputation.id = coeffs.id
+    where other_station_is_local = true
```

Comment:
Since this CTE involves a join, I think it would be clearer to specify the table here:
```suggestion
    where coeffs.other_station_is_local = true
```